types/staking.go

```diff
@@ -100,7 +100,8 @@ type ValidatorSet interface {
 
 	Validator(Context, ValAddress) Validator            // get a particular validator by operator address
 	ValidatorByConsAddr(Context, ConsAddress) Validator // get a particular validator by consensus address
-	TotalPower(Context) Int                             // total power of the validator set
+	TotalBondedTokens(Context) Int                      // total bonded tokens within the validator set
```

Comment:
interested in the rationale behind the interface change here.

Reply:
Power refers to Tendermint Power which is intended to be used nearly nowhere besides for Tendermint (and slashing which needs to interface with tendermint responses) - this interface didn't make a difference pre-Tendermint Power Reduction (https://github.com/cosmos/cosmos-sdk/pull/3400) but now these two things are intended to mean _very_ different values

Reply:
Agreed with @rigelrozanski - this is much clearer, thanks